gravitee-apim-console-webui/src/management/application/details/notifications/notification-settings/notification-settings-list/application-notification-settings-list.module.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { GioLoaderModule } from '@gravitee/ui-particles-angular';
+import { MatTableModule } from '@angular/material/table';
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { UIRouterModule } from '@uirouter/angular';
+import { HttpClientModule } from '@angular/common/http';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
+
+import { ApplicationNotificationSettingsListComponent } from './application-notification-settings-list.component';
+
+import { ApplicationNotificationSettingsDetailsComponent } from '../notification-settings-details/application-notification-settings-details.component';
+import { NotificationSettingsListModule } from '../../../../../../components/notifications/notifications-list/notification-settings-list.module';
+import { GioPermissionModule } from '../../../../../../shared/components/gio-permission/gio-permission.module';
+import { GioTableWrapperModule } from '../../../../../../shared/components/gio-table-wrapper/gio-table-wrapper.module';
+
+@NgModule({
+  declarations: [ApplicationNotificationSettingsListComponent, ApplicationNotificationSettingsDetailsComponent],
+  exports: [ApplicationNotificationSettingsListComponent, ApplicationNotificationSettingsDetailsComponent],
+  imports: [
+    CommonModule,
+    HttpClientModule,
+
+    GioLoaderModule,
+    GioPermissionModule,
+    GioTableWrapperModule,
+
+    MatButtonModule,
+    MatDialogModule,
+    MatIconModule,
+    MatSnackBarModule,
+    MatTableModule,
+    MatTooltipModule,
+
+    UIRouterModule,
+    NotificationSettingsListModule,
+  ],
+})
+export class ApplicationNotificationSettingsListModule {}
```

Comment:
```suggestion
export class ApplicationNotificationSettingsModule {}
```
IMHO this is the main module so for both list & details

Reply:
Done